src/app/cultivating/cultivating-page.component.ts

```diff
@@ -14,9 +14,11 @@ import { SproutsListComponent } from './sprouts-list/sprouts-list.component';
 
       <app-sprouts-list [sprouts]="sprouts() ?? []" (showDetails)="onShowDetails($event)" />
 
-      @if (sproutDetails(); as sprout) {
-        <app-sprout-details-dialog [sprout]="sprout" (close)="sproutDetails.set(null)" />
-      }
+      <app-sprout-details-dialog
```

Comment:
couldn't figure out fast enough how native dialog can work with native/angular animations and with `@if {}` condition around dialog component to get rid of `@if` + `ng-template` for a content